s32/mcux/devices/S32K344/S32K344_features.h

```diff
@@ -200,6 +202,11 @@
 /* @brief Register CHCFGn width. */
 #define FSL_FEATURE_DMAMUX_CHCFG_REGISTER_WIDTH (8)
 
+/* FLEXIO module features */
+
+/* @brief Has doze mode in Flexio */
+#define FSL_FEATURE_FLEXIO_HAS_NO_DOZE_MODE_SUPPORT (1)
```

Comment:
Is it necessary to add all the flexio features even if we are not using them right now ?
Since if we are not using them then we define them as (0), which is as good as not defining them.
As the check condition is always like this -
#if defined(FSL_FEATURE_FLEXIO_HAS_XXX) && FSL_FEATURE_FLEXIO_HAS_XXX